src/main/algorithms/sort/quickSort.ts

```diff
@@ -24,6 +24,22 @@ export class QuickSort implements SortAlgorithmImplementation {
         return pB.build()
     }
 
+    description(): string[] {
+        return ['Quicksort ist ein rekursiver Sortieralgorithmus, der auf dem divide-and-conquer-Prinzip basiert. Die \
```

Comment:
Vielleicht: "Quicksort ist ein rekursiver Algorithmus, das heißt, dass er sich selbst aufruft. Er basiert auf dem Divide-and-Conquer-Prinzip."
Kein Muss, nur eine Idee...

Reply:
das würde ich eher nicht machen, da es ein paar mehr rekursive Algorithmen gibt, die wir haben